spring-boot-latency/README.md

```diff
@@ -0,0 +1,93 @@
+# Introduce latency using the Chaos Monkey for Spring Boot
+
+This experiment explores whether my application can robustly respond to an increase in latency, injected 
+using the [Chaos Monkey for Spring Boot][chaosmonkey].
+
+[chaosmonkey]: https://codecentric.github.io/chaos-monkey-spring-boot/
+
+## Setup for Experiment Execution
+
+### Install the Chaos Toolkit
+
+This experiment uses the [free and open source Chaos Toolkit][chaostoolkit]. The instructions for installing the [Chaos Toolkit][chaostoolkit] are available in the [project's documentation][docs].
+
+[chaostoolkit]: https://chaostoolkit.org/
+[docs]: https://docs.chaostoolkit.org
+
+### Install Any Required Drivers
+
+The chaostoolkit is extended by adding [drivers]. This experiment requires the following drivers:
```

Comment:
chaostoolkit -> Chaos Toolkit and make link

Reply:
Done.